src/main/webapp/examples/team.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description"
+          content="Volunteer driven organisation which empowers students, education institutes and education as a whole in Sri Lanka.">
+    <meta name="author" content="Sustainable Education Foundation">
+    <title>Sustainable Education Foundation</title>
+    <!-- Favicon -->
+    <link href="../assets/img/brand/favicon.png" rel="icon" type="image/png">
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet">
+    <!-- Icons -->
+    <link href="../assets/vendor/nucleo/css/nucleo.css" rel="stylesheet">
+    <link href="../assets/vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet">
+    <!-- Argon CSS -->
+    <link type="text/css" href="../assets/css/argon.css?v=1.0.1" rel="stylesheet">
+    <!-- Global custom CSS -->
+    <link type="text/css" href="../assets/css/global-custom.css" rel="stylesheet">
+
+</head>
+
+<body>
+<header class="header-global">
+    <!--nav bar goes here-->
+</header>
+<!-- to use the dynamic navbar, use this instead of above tag => * <div id="navbar"></div> * -->
+
+<main>
+    <section>
+        <div class="container text-center">
+            <div class="mb-5 text-left h2">
+                <span>Lorem ipsum dolor sit amet</span>
+            </div>
+
+            <div id="teamContent"></div>
+
+            <button id="btn" class="btn btn-link btn-outline-white btn-info">SEE MORE</button>
```

Comment:
Change button id to a unique one.  'btn' might be repeated when using with other pages.

Reply:
Done